druntime/src/core/sys/windows/winsock2.d

```diff
@@ -10,7 +10,7 @@ version (Windows):
 pragma(lib, "ws2_32");
 
 extern(Windows):
-nothrow:
+nothrow @nogc:
 
```

Comment:
as an ignorant reviewer I am totally fine with this...

Reply:
I actually removed that, since the rest of the file was specifying it function-by-function.
I personally think it is appropriate to put `@nogc:` at the top of windows headers though... but I won't make that change here.